src/fiatExchanges/FiatExchangeAmount.tsx

```diff
@@ -153,17 +155,33 @@ function FiatExchangeAmount({ route }: Props) {
       currency,
       flow,
     })
-
-    navigate(Screens.SelectProvider, {
-      flow,
-      selectedCrypto: currency,
-      amount: {
-        crypto: inputCryptoAmount.toNumber(),
-        // Rounding up to avoid decimal errors from providers. Won't be
-        // necessary once we support inputting an amount in both crypto and fiat
-        fiat: Math.round(inputLocalCurrencyAmount.toNumber()),
-      },
-    })
+    const cryptoAmount = inputCryptoAmount.toNumber()
+    // Rounding up to avoid decimal errors from providers. Won't be
+    // necessary once we support inputting an amount in both crypto and fiat
+    const fiatAmount = Math.round(inputLocalCurrencyAmount.toNumber())
+
+    // Navigate straight to the review screen if the user has a FiatConnect fiat account saved
```

Comment:
having a fiat account doesn't mean they'll have passed KYC

Reply:
they might've even tried to cash in with the same provider and not needed to do kyc for the last transfer, but need to do it for this one